CRM/Upgrade/Incremental/schema/5.76.alpha1.SiteToken.entityType.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+return [
+  'name' => 'SiteToken',
+  'table' => 'civicrm_site_token',
+  'class' => 'CRM_Core_DAO_SiteToken',
+  'getInfo' => fn() => [
+    'title' => ts('Site Token'),
+    'title_plural' => ts('Site Tokens'),
+    'description' => ts('Site-wide tokens.'),
+    'add' => 5.76,
+    'log' => TRUE,
+    'icon' => 'fa-code',
+  ],
+  'getPaths' => fn() => [
+    'add' => 'civicrm/admin/sitetoken/edit?action=add&reset=1',
+    'update' => 'civicrm/admin/sitetoken/edit#?SiteToken1=[id]',
+    'browse' => 'civicrm/admin/sitetoken?action=browse&reset=1',
+  ],
+  'getFields' => fn() => [
+    'id' => [
+      'title' => ts('Site Token ID'),
+      'sql_type' => 'int unsigned',
+      'input_type' => 'Number',
+      'required' => TRUE,
+      'primary_key' => TRUE,
+      'auto_increment' => TRUE,
+    ],
+    'domain_id' => [
+      'title' => ts('Domain'),
+      'sql_type' => 'int unsigned',
+      'input_type' => 'EntityRef',
+      'required' => TRUE,
+      'entity_reference' => [
+        'entity' => 'Domain',
+        'key' => 'id',
+        'on_delete' => 'CASCADE',
+      ],
+      'pseudoconstant' => [
+        'table' => 'civicrm_domain',
+        'key_column' => 'id',
+        'label_column' => 'name',
+      ],
+    ],
+    'name' => [
+      'title' => ts('Token Name'),
+      'sql_type' => 'varchar(64)',
+      'input_type' => 'Text',
+      'description' => ts('Token string, e.g. {site.[name]}'),
+      'input_attrs' => [
+        'maxlength' => 64,
+      ],
+    ],
+    'label' => [
+      'title' => ts('Token Label'),
+      'sql_type' => 'varchar(255)',
+      'input_type' => 'Text',
+      'description' => ts('User-visible label in token UI'),
+      'input_attrs' => [
+        'label' => ts('Label'),
+        'maxlength' => 255,
+      ],
+    ],
+    'body_html' => [
+      'title' => ts('Token Value (HTML)'),
+      'sql_type' => 'text',
+      'input_type' => 'TextArea',
+      'description' => ts('Value of the token in html format.'),
+      'input_attrs' => [
+        'rows' => 8,
+        'cols' => 80,
+      ],
+    ],
+    'body_text' => [
+      'title' => ts('Token Value (Text)'),
+      'sql_type' => 'text',
+      'input_type' => 'TextArea',
+      'description' => ts('Value of the token in text format.'),
```

Comment:
I wonder if we can make it clear that this may not be needed as it will be inferred if not present